client/modules/workspace/src/components/SystemStatusModal/SystemStatusModal.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { Modal, Spin, Alert } from 'antd';
+import { SystemQueueTable } from './SystemQueueTable';
+import { useSystemOverview } from '@client/hooks';
+import styles from './SystemStatusModal.module.css';
+
+interface SystemStatusModalProps {
+  isModalVisible: boolean;
+  onClose: () => void;
+}
+
+export const SystemStatusModal: React.FC<SystemStatusModalProps> = ({
+  isModalVisible,
+  onClose,
+}) => {
+  const [activeSystem, setActiveSystem] = useState('frontera');
```

Comment:
The default system should be derived from the selected app. You can use the `useGetAppsSuspense` hook